frontend/src/pages/notebookController/screens/server/GPUSelectField.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from 'react';
-import { FormGroup, Select, SelectOption, Skeleton } from '@patternfly/react-core';
-import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { FormGroup, Select, SelectOption, Skeleton, Popover } from '@patternfly/react-core';
```

Comment:
This field is not even used anymore, this is not the coponent you are looking for.
You might wanna take a look at `AcceleratorSelectField.tsx`, that's where the accelerator logic is placed.

Btw, @kaedward @vconzola regarding this text https://www.sketch.com/s/113593f8-5970-49d6-a352-709b07639127/a/ZVnwmoZ we might wanna take a look cause we are using now accelerators, not sure if we want to change the wording here

Reply:
@lucferbux thanks, How do we enable Accelerator?

Reply:
Hi, for this you either should have a GPU enabled cluster, or in a PSI cluster, you can pretty much deploy an accelerator resource like this in your cluster:

```yaml
kind: AcceleratorProfile
apiVersion: dashboard.opendatahub.io/v1
metadata:
  name: migrated-gpu
  namespace: opendatahub
spec:
  displayName: Nvidia GPU
  identifier: nvidia.com/gpu
  enabled: true
  tolerations:
    - effect: NoSchedule
      key: nvidia.com/gpu
      operator: Exists
```

And with that, once you refresh, you will get the GPU field.